lib/foreman_rh_cloud.rb

```diff
@@ -1,4 +1,30 @@
 require 'foreman_rh_cloud/engine.rb'
 
 module ForemanRhCloud
+  module WebUi
+    ADVISOR = 'advisor'
+    VULNERABILITY = 'vulnerability'
+    PATCH = 'patch'
+
+    def self.insights_system_url(subsystem, host_uuid)
+      ForemanRhCloud.base_url + "/insights/#{subsystem}/systems/#{host_uuid}"
+    end
+  end
+  def self.base_url
+    # for testing set ENV to 'https://ci.cloud.redhat.com'
+    @base_url ||= ENV['SATELLITE_INVENTORY_CLOUD_URL'] || 'https://cloud.redhat.com'
+  end
+
+  def self.authentication_url
+    # https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token
+    @authentication_url ||= ENV['SATELLITE_INVENTORY_CLOUD_SSO_URL'] || 'https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token'
+  end
+
+  def self.hits_export_url
+    base_url + '/api/insights/v1/export/hits/'
+  end
```

Comment:
This should move to InsightsCloud module - it is specific to insights hits handling, but should consume general RH cloud base URL:

```suggestion
  def self.hits_export_url
    ForemanRhCloud.base_url + '/api/insights/v1/export/hits/'
  end
```